index.js

```diff
@@ -378,11 +378,13 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = ver || msg || opt || undefined
```

Comment:
How do y'all feel about defaulting the `version` value to the parent package.json value (if found)? It has always bugged me that I have to code this manually each time, and I always end up doing the same thing. Am I just missing something?